lms/envs/common.py

```diff
@@ -1858,6 +1858,8 @@
     'provider.oauth2',
     'oauth2_provider',
 
+    'third_party_auth',
```

Comment:
Is this going to change the requirements for people who aren't interested in third party auth? Maybe we don't want to have it installed all the time?